test/e2e/operator.go

```diff
@@ -436,18 +436,26 @@ var _ = Describe("ARO Operator - ImageConfig Reconciler", func() {
 
 	// Reimplementation of function from image config controller
 	getCloudAwareRegistries := func(instance *arov1alpha1.Cluster) ([]string, error) {
-		var requiredRegistries []string
+		const replicationRegistryFormat = "%s.%s.data.%s"
```

Comment:
Personally I am good with using this format string in-line with your `replicationRegistry = fmt.Sprintf(...`. Declaring `replicationRegistryFormat` here and then only using it once in the function might be unnecessary.